crates/lib/src/services/minetest_master_server/service.rs

```diff
@@ -0,0 +1,13 @@
+use crate::http::HttpClient;
+use crate::minetest_master_server::types::Response;
+use crate::{GDResult, TimeoutSettings};
+
+pub fn query(timeout_settings: TimeoutSettings) -> GDResult<Response> {
+    let mut client = HttpClient::from_url(
```

Comment:
This won't work without TLS (returns a 400 request error because will try to send a HTTP request to port 443). Perhaps feature gate this behind TLS. (although I do think that is a bad HttpClient behaviour).

The service does also work over HTTP so another option would always be using HTTP. Or use HTTPS if tls feature is enabled and HTTP if its not.